user-operator/src/main/java/io/strimzi/operator/user/Main.java

```diff
@@ -70,13 +72,15 @@ public static void main(String[] args) {
         UserOperatorConfig config = UserOperatorConfig.fromMap(System.getenv());
         KubernetesClient client = new OperatorKubernetesClientBuilder("strimzi-user-operator", Main.class.getPackage().getImplementationVersion()).build();
         Admin adminClient = createAdminClient(config, client, new DefaultAdminClientProvider());
+        ExecutorService kafkaUserOperatorExecutor = Executors.newFixedThreadPool(config.getUserOperationsThreadPoolSize(), r -> new Thread(r, "operator-thread-pool"));
```

Comment:
Doesn't this mean all the threads will have the same name? IF so, that could result in some confusing logs. Better to use names which are more likely to be unique (e.g. using an `AtomicInteger` to generate a more unique name on thread creation.